pkg/session/bootstrap.go

```diff
@@ -583,6 +583,7 @@ const (
 		meta LONGBLOB,
 		concurrency INT(11),
 		step INT(11),
+		target_scope VARCHAR(256),
```

Comment:
```suggestion
		target_scope VARCHAR(256) default "",
```
for previous version tasks set it to "", to avoid check IsNull in Row2Task